axelrod/strategies/selfsteem.py

```diff
@@ -0,0 +1,53 @@
+from axelrod.actions import Action, Actions
+from axelrod.player import Player
+
+from math import pi, sin
+import random
+
+C, D = Actions.C, Actions.D
+
+
+class SelfSteem(Player):
+    """
+    This strategy is based on the feeling with the same name.
+    It is modeled on the sine curve(f = sin( 2* pi * n / 10 )), which varies
+    with the current iteration.
+    
+    If f > 0.95, 'ego' of the algorithm is inflated; always defects.
+    If 0.95 > |f| > 0.3, rational behavior; follows TitForTat algortithm.
+    If 0.3 > f > -0.3; random behavior.
+    If f < -0.95, algorithm is at rock bottom; always cooperates.
+    
+    Names:
+    
+    - SelfSteem: [Andre2013]_
+    """
+
+    name = 'SelfSteem'
```

Comment:
It is `SelfSteem`. I mistakenly named the test `test_selfesteem` instead of `test_selfsteem`.